master/getting-started/docker/tutorials/advanced-policy.md

```diff
@@ -47,184 +42,158 @@ container to network AB.
 
 ## Managing Calico policy for a network
 
-This section walks through an example of creating a docker network (with
+This section walks through two examples of creating a docker network (with
 Calico) and using the `calicoctl` command line interface to modify the policy
-associated with that network.
+associated with that network.  One example applies policy directly through the
+profile, the other applies policy through label selection.  Both approaches
+may be used together - in which case the selector based policy is applied first.
+
+### Global policy applied through label selection
 
-#### Create a Docker network
+Suppose we want to provide network policy for a set of database containers and a
+set of app containers, restricting the connectivity to allow only
+TCP connections from the app containers to the database containers connecting to 
```

Comment:
I think this should go right before the "Global policy applied" title instead of after it. It'd be more clear to introduce the scenario, then show how each policy approach enforces it